lib/anagrams.rb

```diff
@@ -1,2 +1,7 @@
 def anagrams?(str1, str2)
-end
\ No newline at end of file
+if str1.sum == str2.sum then
+       puts 'True' 
```

Comment:
`puts "True"` is a lot like `puts "Banana"`: It's just displaying a string. There are two things wrong with this:

1. The goal of the method is not to _display_ text. It's to _return_ a value. So `return "True"` would be closer to correct.
2. You shouldn't be returning a _string_ at all. `"True"` is like `"Banana"` in that they're both just sets of characters. But remove the quotes and lowercase the 'T' and you get `true` (no quotes). _That_ is not a string at all. It's a **boolean**.